packages/core/admin/admin/src/core/store/configure.ts

```diff
@@ -25,10 +22,7 @@ const staticReducers = {
   rbacProvider: RBACReducer,
   'content-manager_app': cmAppReducer,
   [contentManagerApi.reducerPath]: contentManagerApi.reducer,
-  'content-manager_listView': listViewReducer,
-  'content-manager_rbacManager': rbacManagerReducer,
-  'content-manager_editViewLayoutManager': editViewReducer,
-  'content-manager_editViewCrudReducer': crudReducer,
+  'content-manager': contentManagerReducers,
```

Comment:
Should content-manager_app be part of this?

Reply:
I'll probably do this in a separate PR, only because this is already massive – but you're right, it should & will be.